spec/helpers/simple_server_env_helper_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe SimpleServerEnvHelper do
+  include SimpleServerEnvHelper
+
+  let(:simple_server_env) {"SIMPLE_SERVER_ENV"}
+
+  after(:each) do
+    ENV[simple_server_env] = "test"
+  end
+
+  describe 'style_class_for_environment' do
```

Comment:
Might be better to contextualize these specs within multiple `it`s per environment. Simpler to keep one assertion per spec.

Reply:
Added contexts per environment.